bazel-amilusSelenium

```diff
@@ -0,0 +1 @@
+/private/var/tmp/_bazel_tcamiluslu/4ea50276b8925b7e467cb75929e42874/execroot/selenium
```

Comment:
This change is not related to the PR, remove it from this PR.

Reply:
Yeap, I just deleted this file from PR.